simpleocr/ocr.py

```diff
@@ -1,10 +1,10 @@
 import numpy
-from opencv_utils import show_image_and_wait_for_key, draw_segments
-import segmentation as segmenters
-import classification as classifiers
-import feature_extraction as extractors
-import grounding as grounders
-from files import ImageFile
+from .opencv_utils import show_image_and_wait_for_key, draw_segments
+import simpleocr.segmentation as segmenters
```

Comment:
Is there a reason you opted to use package imports here instead of the relative imports you use elsewhere?